package-res/resources/web/prompting/api/EventAPI.js

```diff
@@ -79,13 +79,35 @@ define([], function() {
      * @method
      * @param {Function} callback   The function to be executed when the event is triggered.
      *                              Pass null if you wish to unbind this event
-     * @example
+     * @param {String} paramName    The name of the parameter to which the callback will be called
+     *                              This argument is optional. If no parameter name is passed on
+     *                              the callback will be assigned to all the parameters.
+     *
+     * @example <caption>Register a parameter changed event to all parameters</caption>
      *  api.event.parameterChanged(function(parameterName, parameterValue) {
      *    // Execute event based code
      *  })
+     *
+     * @example <caption>Register a parameter changed event to the parameter called "parameterName"</caption>
+     *  api.event.parameterChanged(function(parameterName, parameterValue) {
+     *    // Execute event based code
+     *  }, "parameterName")
      */
-    this.parameterChanged = function(callback) {
-      api.operation._getPromptPanel().onParameterChanged = callback;
+    this.parameterChanged = function(callback, paramName) {
+      if (typeof callback === 'function') {
+        onParameterChanged = api.operation._getPromptPanel().onParameterChanged;
+        if (!paramName || paramName == '') {
```

Comment:
In this statement enough using only first if (!paramName) because it also checks situation when paramName is empty.